src/servers/ZoneServer2016/entities/character.ts

```diff
@@ -151,6 +151,7 @@ export class Character2016 extends BaseFullCharacter {
   ) => void;
   timeouts: any;
   hasConveys: boolean = false;
+  hasBoots: boolean = false;
```

Comment:
Since now there is hasConveys and hasBoots, refactoring to smth like `shoesType: ShoesTypeEnum` but this can be done later on

Reply:
Yea for sure! I'll look into that once I get Conveys to lose durability in some way 👍